src/gl/directx11/vcTexture.cpp

```diff
@@ -421,7 +473,7 @@ bool vcTexture_EndReadPixels(vcTexture *pTexture, uint32_t x, uint32_t y, uint32
   pPixelData = ((uint32_t *)msr.pData) + (x + y * pTexture->width);
   for (int i = 0; i < (int)height; ++i)
   {
-    memcpy((uint8_t*)pPixels + (i * pTexture->width), pPixelData, width * pixelBytes);
+    memcpy((uint8_t *)pPixels + (i * pTexture->width), pPixelData, width * pixelBytes);
```

Comment:
There are quite a lot of these throughout this MR- I think your VS settings are set to "right only" instead of "leave alone"